src/flap.sol

```diff
@@ -49,24 +49,24 @@ contract Flapper is LibNote {
 
     // --- Data ---
     struct Bid {
-        uint256 bid;
-        uint256 lot;
+        uint256 bid;  // gems paid     [wad]
+        uint256 lot;  // dai for sale  [rad]
         address guy;  // high bidder
         uint48  tic;  // expiry time
```

Comment:
`tic` & `end` should have comments similar to those suggested below for the `Flipper`